env.example

```diff
@@ -271,7 +271,7 @@ JICOFO_AUTH_USER=focus
 #JICOFO_RESERVATION_REST_BASE_URL=http://reservation.example.com
 
 # Enable Jicofo's health check REST API (http://<jicofo_base_url>:8888/about/health)
-#JICOFO_ENABLE_HEALTH_CHECKS=true
+JICOFO_ENABLE_HEALTH_CHECKS=false
```

Comment:
Please revert all changes to the env.example file, I want to move all of those to the handbook instead.

Reply:
Removed new env variables from env.example file.